src/data/markdown/docs/03 cloud/01 Creating and running a test/999 Troubleshooting.md

```diff
@@ -226,16 +226,27 @@ If you need to increase this limit, contact our support team.
 Additionally, you can change the concurrency limit policy that defines how k6 Cloud acts when the organization reaches the limit and a new test run is triggered.
 Two options are available:
 
-- `Abort test`: the new test run will be automatically aborted. This is the default option.
-- `Queue test`: the new test run will be queued for execution and started once a slot is opened. A queued test will timeout if no slot is available in 6 hours.
+- `Queue test`: the new test run will be queued for execution and started once a slot is opened. A queued test will timeout if no slot is available in 6 hours. This is the current default option.
+- `Abort test`: the new test run will be automatically aborted.
+
+<Blockquote mod="note" title="Concurrency limit policy default change">
+
+On 31.01.2023, the default concurrency policy changed from `Abort test` to `Queue test`.
+To avoid confusion, k6 kept the old default for organizations who joined before the change.
+
+If your organization uses the old default and wants to upgrade to `Queue test` policy, simply follow the instructions below.
+
+</Blockquote>
```

Comment:
```suggestion
<Blockquote mod="note" title="Changes to the default concurrency limit policy">

On January 1st, 2023, the default concurrency policy changed from `Abort test` to `Queue test`.
To avoid confusion, k6 kept the old default for organizations who joined before the change.

If your organization uses the old default and wants to upgrade to `Queue test` policy,change the policy at **Organization settings > Settings**.

</Blockquote>
```

Reply:
Explanation of my notes:

- It can be hard to parse a huge stack of adjectives, so I add a preposition and removed a word
- Dates formats will confuse different audiences. Better to write it (Or use ISO, at least :-) ).
- It seems like the instructions are just one line, we can be a little redundant. IMO better UX than directing user to a new section.

Reply:
If these look okay to you, it's fine to me.